DEVDOCS-3922: [net new] Create POS Foundation overview (#1554)
bigcommerce/dev-docs: docs/api-docs/partner/pos/pos-foundation-overview.md

# POS Foundation

> POS Foundation provides a proof-of-concept [open-source framework](https://github.com/bigcommerce/point-of-sale-foundation) that helps developers deliver custom point-of-sale (POS) checkout interfaces to BigCommerce merchants with brick-and-mortar locations. It provides a template to scaffold custom POS solutions that use secure, pre-certified EMV card readers. POS Foundation's default integration with Stripe Terminal helps significantly accelerate development time. 

Reviewer: ```suggestion
POS Foundation provides a proof-of-concept [open source framework](https://github.com/bigcommerce/point-of-sale-foundation) for building point-of-sale (POS) checkout interfaces that serve BigCommerce merchants with physical locations. It provides a template to scaffold custom POS solutions that use secure, pre-certified EMV card readers. POS Foundation's default integration with Stripe Terminal can significantly accelerate development time. 
```

> POS Foundation is a manual connector app that uses a store API account. The app will not be displayed in the store control panel. POS Foundation does not require that you create an app profile or use ngrok. This proof-of-concept tutorial only applies to local network operations and in its current state does not create its own channel. We are looking to expand this foundation and would like to collaborate with interested agencies.

Reviewer: ```suggestion
POS Foundation scaffolds manual connector apps that use [store API accounts](/api-docs/getting-started/authentication/rest-api-authentication#store-api-accounts). You do not need to create an app profile or use ngrok. The app will not appear in the store control panel and is only accessible to computers running on the same local network as the app's server. In contrast to [Subscription Foundation](/api-docs/partner/subscription-solutions/foundation-guide)-derived apps, POS Foundation apps do not create their own dedicated sales channels.  If you're interested in contributing to this package, >>> what did we want them to do? submit a PR? contact partner rep?

```

## Software requirements
* [Node.js](https://nodejs.org/en/) 14.0.0+
* The [npm](https://www.npmjs.com/) package manager
* A MongoDB database instance; this guide uses [MongoDB Cloud](https://www.mongodb.com/cloud)

## Hardware requirements
* A [Stripe Terminal-enabled card reader](https://stripe.com/terminal)
* A card reader-compatible device to run the POS system

## Configure accounts

<!-- theme: info -->
> #### Account configuration requirements
> We recommend that you use a sandbox store that has the same multi-storefront status as the production store. For example, if you're developing for a multi-storefront-enabled merchant store, use a multi-storefront sandbox. For information on configuring multi-storefront, see [Multi-Storefront](https://support.bigcommerce.com/s/article/Multi-Storefront?language=en_US).

To configure your accounts, complete the following steps:

> 1. [Create a BigCommerce Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation) to develop and test your apps.

Reviewer: ```suggestion
1. To develop and test safely, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=pos-foundation).
```

Reviewer: parallel with Subscription Foundation

4. Access the database to verify you have created the collections and documents correctly. You can use the following command, or connect with a GUI, such as [Compass](https://www.mongodb.com/products/compass).

```shell title="Verify migration and seed"
npx prisma studio
``` 

5. To start the server that runs the POS system, run the following command:

```shell title="Start the server"
npm run dev
```
Upon successfull execution of this command, the app is running locally.

6. Sign in to the app as an admin at `http://localhost:3000/signin`. The default admin PIN in the seed data is `1234`.

![pos-sign-in](https://storage.googleapis.com/bigcommerce-production-dev-center/images/pos-sign-in.png)

7. After you access Prisma Studio, navigate to the **Settings** screen to save your store address, which the app uses for checkout tax calculations. If you fail to supply the store address, you will receive the following error:
  
![pos-missing-store-address](https://storage.googleapis.com/bigcommerce-production-dev-center/images/POS-missing-store-address.jpeg)

> ## Connecting to a reader in the POS

Reviewer: linked to stripe reference. done.
